backend/app/src/main/java/com/moh/phlat/backend/service/ProcessDataServiceImpl.java

```diff
@@ -3,10 +3,22 @@
 import java.util.ArrayList;
 import java.util.List;
 import com.moh.phlat.backend.service.dto.ReportSummary;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.criteria.CriteriaBuilder;
+import jakarta.persistence.criteria.CriteriaQuery;
+import jakarta.persistence.criteria.Root;
+
 import com.moh.phlat.backend.service.RowStatusService;
```

Comment:
could see some unused imports

Reply:
The three highlighted imports are being used in the getUniqueColumnValues method.